src/main/java/com/epam/petclinic/order/config/JwtConfiguration.java

```diff
@@ -0,0 +1,42 @@
+package com.epam.petclinic.order.config;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.core.io.ClassPathResource;
+import org.springframework.core.io.Resource;
+import org.springframework.security.oauth2.provider.token.TokenStore;
+import org.springframework.security.oauth2.provider.token.store.JwtAccessTokenConverter;
+import org.springframework.security.oauth2.provider.token.store.JwtTokenStore;
+import org.springframework.util.FileCopyUtils;
+
+import java.io.IOException;
+import java.io.InputStream;
+
+@Configuration
+public class JwtConfiguration {
+    @Autowired
+    private JwtAccessTokenConverter jwtAccessTokenConverter;
+
+
+    @Bean
+    @Qualifier("tokenStore")
+    protected TokenStore tokenStore() {
+        return new JwtTokenStore(jwtAccessTokenConverter);
+    }
+
+    @Bean
+    protected JwtAccessTokenConverter jwtTokenEnhancer() {
```

Comment:
Do we really need defining JwtAccessTokenConverter as a `@Bean`? Is it used outside JwtConfiguration?